### DISCUSSION TITLE
中州韵98五笔助手

### DISCUSSION THREAD

**Discussion by yanhuacuo** [rime/home#998] [Show and tell]
## 程序简介

中州韵98五笔助手基于 `C++` ，借助优秀的 `Qt` 框架，实现了跨 `Windows` 、`Linux`，`MacOS` 三种桌面系统。定向处理基于 `98五笔小组` 实现的形码方案。

集成 `输入法行为控制` 与 `码表管理` 功能，当然还有非常理想的 `数据备份方案`。采用多线程模式驱动码表管理器，力求胜任百万行级别的词库处理需求。

目前处于功能诉求整理阶段，开发团队缺少对前端设计或 `QStyle` 样式表较为了解的朋友，只能仓促设计一个较为简陋的简面。希望有趣朋友联系我们，参与这个有趣的项目。

![qrcode_1710736727216](https://github.com/rime/home/assets/3107994/2e681d46-cf32-4dc5-a0b4-464b5dac91dd)

## 图示

当前开发主要在 `Linux` 下完成，初步跨平台编译过，适应性良好。我们一起努力，完善 `RIME` 的生态。该工具完善后，将会解决掉一部分形码用户的上手成本问题。

撒花，项目基本完结啦！

- ![PIC](https://raw.githubusercontent.com/yanhuacuo/fcitx5-wubi98/main/1108/0.png)

- ![PIC](https://raw.githubusercontent.com/yanhuacuo/fcitx5-wubi98/main/1108/1.png)

- ![PIC](https://raw.githubusercontent.com/yanhuacuo/fcitx5-wubi98/main/1108/2.png)

- ![PIC](https://raw.githubusercontent.com/yanhuacuo/fcitx5-wubi98/main/1108/3.png)

- ![PIC](https://raw.githubusercontent.com/yanhuacuo/fcitx5-wubi98/main/1108/4.png)

- ![PIC](https://raw.githubusercontent.com/yanhuacuo/fcitx5-wubi98/main/1108/5.png)

- ![PIC](https://raw.githubusercontent.com/yanhuacuo/fcitx5-wubi98/main/1108/6.png)

- ![PIC](https://raw.githubusercontent.com/yanhuacuo/fcitx5-wubi98/main/1108/7.png)

- ![PIC](https://raw.githubusercontent.com/yanhuacuo/fcitx5-wubi98/main/1108/8.png)

**Comment by yanhuacuo** [comment]
欢迎建言：

 [五笔小筑](https://wubi98.gitee.io/2022/01/15/2022-01-15-001.wb98/)

**Comment by lotem** [comment]
有實力啊。

**Comment by yanhuacuo** [comment]
中州韵98五笔助手近日更新汇总：

1、UI更新；
2、添加方案选单切换功能
3、添加自动加载功能
4、添加使用指导

windows版 、linux版，同步更新

下载 ：http://98wb.ysepan.com/



![Screenshot_20240318_044730](https://github.com/rime/home/assets/3107994/e893342a-5b03-41e7-902c-f2ff1c308e02)
![Screenshot_20240318_044747](https://github.com/rime/home/assets/3107994/2cfcb8d2-5b30-45b7-b8c0-06fea44d5a84)
![Screenshot_20240318_044747-1](https://github.com/rime/home/assets/3107994/dd5d4bb6-aff0-4ea0-898c-73ae665da545)
![Screenshot_20240318_044830](https://github.com/rime/home/assets/3107994/e3474015-a001-4d03-a04f-908602bb5b31)
![Screenshot_20240318_044837](https://github.com/rime/home/assets/3107994/bdf39b84-c8a2-4207-bcea-a5a49a6e2abe)
![Screenshot_20240318_044847](https://github.com/rime/home/assets/3107994/d1e0daae-da9f-4ed0-a6e7-1e97d9a27469)
![Screenshot_20240318_044900](https://github.com/rime/home/assets/3107994/e39fda8f-e00d-4674-ac8f-0249f33a80af)
![Screenshot_20240318_044943](https://github.com/rime/home/assets/3107994/80000a6f-c369-4ef0-9ca9-f7f07c02d86c)
![Screenshot_20240318_044952](https://github.com/rime/home/assets/3107994/2c627739-15ed-441a-8ab1-7e578a6943af)
![Screenshot_20240318_045137](https://github.com/rime/home/assets/3107994/9ad2cf0a-d983-4732-b77d-d5dfc6d9f56f)

**Comment by yanhuacuo** [comment]
视频介绍：

https://www.bilibili.com/video/BV1CZ421t7Tc/?spm_id_from=333.999.0.0
